services-core/src/main/java/com/mapbox/core/utils/TextUtils.java

```diff
@@ -86,117 +85,4 @@ public static String formatCoordinate(double coordinate, int precision) {
     df.setRoundingMode(RoundingMode.FLOOR);
     return df.format(coordinate);
   }
-
-  /**
-   * Used in various APIs to format the user provided radiuses to a String matching the APIs format.
-   *
-   * @param radiuses a double array which represents the radius values
-   * @return a String ready for being passed into the Retrofit call
-   * @since 3.0.0
-   */
-  public static String formatRadiuses(double[] radiuses) {
```

Comment:
We cannot remove these methods as that would break semver.

Reply:
fixed